### PR TITLE
Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by ceohockey60**
Created Maintainers.md

## What have you changed? (mandatory)

Added Maintainers.md to create more transparency of project leadership to community. This is necessary for TiKV's next step growth in the community.

## What are the type of the changes? (mandatory)

Improvement.

## How has this PR been tested? (mandatory)

N/A

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

N/A

## Does this PR affect tidb-ansible update? (mandatory)

N/A

### CODE REVIEW

**Comment by breeswish** [comment]
The format seems not good. I have seen some great samples, like [[1]](https://github.com/chef/chef/blob/master/MAINTAINERS.md), [[2]](https://github.com/habitat-sh/builder/blob/master/MAINTAINERS.md) and [[3]](https://github.com/golang/dep/blob/master/MAINTAINERS.md). Also, make `@Xxx` link to a GitHub profile might be better, because they are not twitter accounts.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Maintainers of this repository and their focus areas:
+
+    Siddon Tang, tl@pingcap.com; @SiddonTang: Project Lead, Raft, Coprocessor, Transaction, Storage, Roadmap
```
Comment (Hoverbear) [inline MAINTAINERS.md]: If you remove the `   ` (you start preformatted text here) at the start here (Maybe `>` instead?) it should link to their accounts for you.

**Comment by ceohockey60** [comment]
Re-formatted the document based on @breeswish suggestions. PTAL.

**Comment by breeswish** [comment]
LGTM

**Comment by gregwebs** [comment]
Did we consider using the CODEOWNERS file supported by github? It integrates to suggest reviewers for pull requests, etc. https://blog.github.com/2017-07-06-introducing-code-owners/

**Comment by breeswish** [comment]
@gregwebs Good idea! It should be very useful.

**Pull request merged**